src/Void/Init.fs

```diff
@@ -5,6 +5,19 @@ open Void.Lang.Interpreter
 open Void.Lang.Editor
 open Void.ViewModel
 
+type VoidOptions = {

```

Comment:
Long-term, this type would become huge. Like my comments above, it's fine for now, but keep your eye on it to break it down as we add command-line arguments. The strategy outlined above would eliminate the need for a mammoth type such as this entirely. That is part of the advantage of the service/event-driven model.